src/components/Truncate/Truncate.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
+import TruncateMarkup from 'react-truncate-markup';
+
+// todo - crum: this component doesn't currently support 'className' or 'style' because 3rd party 'TruncateMarkup' doesn't support it...use TypeScript 'omit???' Omit<IXProps, "unwantedProp">
```

Comment:
Calling this todo out

Reply:
I'll go through and check the repo for all `todos`.